coffee/slide.coffee

```diff
@@ -13,7 +13,8 @@ document.addEventListener 'DOMContentLoaded', ->
       for target in $(@).attr('data-marp-path-resolver').split(/\s+/)
         $(@).attr(target, resolvePathFromMarp($(@).attr(target)))
 
-    Markdown = new clsMarkdown({ afterRender: clsMarkdown.generateAfterRender($) })
+    twemojiBasePath = Path.resolve(__dirname, '../images/twemoji/') + '/'
```

Comment:
...and the separator should use `Path.sep` instead of `/` to output correct path in Windows.